src/lib.rs

```diff
@@ -197,8 +197,14 @@ impl vm::Ext for EwasmExt {
 
     /// Returns code at given address
     fn extcode(&self, address: &Address) -> Result<Option<Arc<Bytes>>> {
-        // FIXME: implement
-        unimplemented!()
+        let address = &Bytes20 { bytes: address.0 };
+        let size = ewasm_api::external_code_size(address);
+        let mut code = vec![0; size];
+        let res = ewasm_api::external_code_copy(address, 0, size, &mut code);
```

Comment:
You can use `external_code_acquire` here.

Reply:
updated